### PR TITLE
Icon widget refactor

### PR DESCRIPTION
**Pull request by FinalAngel**
This cleans up the code for the icon widget and centralises the javascript to base.js

image

image

### CODE REVIEW

**Comment by intellectronica** [comment]
:+1:

File: aldryn_bootstrap3/static/aldryn_bootstrap3/js/base.js [hunk 1]
```diff
@@ -22,17 +22,18 @@
...
              * @method contextWidget
+             * @param {jQuery} element context element to render
              */
-            contextWidget: function contextWidget() {
-                var data = $('.aldryn-bootstrap3-context').data();
+            contextWidget: function contextWidget(element) {
+                var data = element.data();
                 var fieldName = data.context;
-                var contextInputs = $('.js-btn-group-context-' + fieldName + ' label input');
+                var contextInputs = element.find('.js-btn-group-context-' + fieldName + ' label');
```
Comment (vxsx) [inline aldryn_bootstrap3/static/aldryn_bootstrap3/js/base.js]: variable name is a bit misleading

**Comment by vxsx** [comment]
LGTM

File: aldryn_bootstrap3/static/aldryn_bootstrap3/js/base.js (hunk 1 above)
Comment (FinalAngel) [inline aldryn_bootstrap3/static/aldryn_bootstrap3/js/base.js]: it is if you check the backend code

**Pull request merged**